### ISSUE TITLE
Exception generated

### ISSUE THREAD

**Issue by tight-git** [regosen/gallery_get#75]
robin@robin-linux:~/robin/Porn/Fusker/ImageFap/gallery_get-master$ python gallery_get.py
Input URL: https://www.imagefap.com/pictures/8801028/Nude-outdoor-221-%2F-Nackt-im-Freien-221
Destination (8800994): 8801028
Using imagefap plugin...

------------------------------------------------------------
Traceback (most recent call last):
  File "gallery_get.py", line 453, in run_wrapped
    root = GalleryGet(myurl, dest or DEST_ROOT, titleAsFolder, allowGenericPlugin).run()
  File "gallery_get.py", line 444, in run
    return self.queue_jobs(page, root, subtitle)
  File "gallery_get.py", line 379, in queue_jobs
    links = run_match(self.plugin.redirect, page)
  File "gallery_get.py", line 115, in run_match
    result = match(source)
  File "/home/robin/robin/Porn/Fusker/ImageFap/gallery_get-master/gallery_plugins/plugin_imagefap.py", line 19, in redirect
    cur_url = re.findall(r'href=\'(.+?)\'\>', source)[0].split("?")[0]
IndexError: list index out of range
Using params: ['https://www.imagefap.com/pictures/8801028/Nude-outdoor-221-%2F-Nackt-im-Freien-221', '8801028', False]
------------------------------------------------------------

An exception occurred!  We can help if you follow these steps:

1. Visit https://github.com/regosen/gallery_get/issues
2. Click the "New Issue" button
3. Copy the text between the lines and paste it into the issue.
(If you don't want to share the last line, the rest can still help.)
robin@robin-linux:~/robin/Porn/Fusker/ImageFap/gallery_get-master$ 
robin@robin-linux:~/robin/Porn/Fusker/ImageFap/gallery_get-master$ 
robin@robin-linux:~/robin/Porn/Fusker/ImageFap/gallery_get-master$ python gallery_get.py
Input URL: https://www.imagefap.com/pictures/8801028/Nude-outdoor-221-%2F-Nackt-im-Freien-221
Destination (8801028): 8801028
Using imagefap plugin...

------------------------------------------------------------
Traceback (most recent call last):
  File "gallery_get.py", line 453, in run_wrapped
    root = GalleryGet(myurl, dest or DEST_ROOT, titleAsFolder, allowGenericPlugin).run()
  File "gallery_get.py", line 444, in run
    return self.queue_jobs(page, root, subtitle)
  File "gallery_get.py", line 379, in queue_jobs
    links = run_match(self.plugin.redirect, page)
  File "gallery_get.py", line 115, in run_match
    result = match(source)
  File "/home/robin/robin/Porn/Fusker/ImageFap/gallery_get-master/gallery_plugins/plugin_imagefap.py", line 19, in redirect
    cur_url = re.findall(r'href=\'(.+?)\'\>', source)[0].split("?")[0]
IndexError: list index out of range
Using params: ['https://www.imagefap.com/pictures/8801028/Nude-outdoor-221-%2F-Nackt-im-Freien-221', '8801028', False]
------------------------------------------------------------

**Comment by regosen** [comment]
Looks like there are no longer any links available on that gallery.  I just made a change to handle this case more elegantly.

If you try again with my latest changes, it should no longer throw an exception for you.

**Comment by tight-git** [comment]
Thank you
   After submitting the report, I did notice that the gallery didnt seem to be there any longer. Being used to programming in Ada, did think exception raised was a little harsh - so happy.

    On Monday, 1 June 2020, 15:58:29 UTC, Rego Sen <notifications@github.com> wrote:  
 
 


Looks like there are no longer any links available on that gallery. I just made a change to handle this case more elegantly.

If you try again with my latest changes, it should no longer throw an exception for you.
